great-idea-website/index.html

```diff
@@ -14,50 +14,101 @@
   <![endif]-->
 </head>
 
+<!-- I did it!!! -->
   <body>
-
-  Services
-  Product
-  Vision
-  Features
-  About
-  Contact
+  
+ <div ="navTop">
+  <header>
+  
+<nav>
+  <ul>
+  <li><a href="#">Services</a> </li>
+  <li><a href="#">Product</a></li>
+  <li><a href="#">Vision</a></li>
+  <li><a href="#">Features</a></li>
+  <li><a href="#">About</a></li>
+  <li><a href="#">Contact</a></li>
+  </ul>
+</nav>
 
   <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-  Innovation
-  On
-  Demand
-
+  
+  </header>
+  
+ </div>
+ 
+ <div class="container">
+  
+  
+  <div class="sectionOne">
```

Comment:
Try to avoid using class names with one, two, first, second, etc. It's more descriptive to use a name like "top-section" or "tagline-section."